src/python/impactx/plot/Survey.py

```diff
@@ -128,7 +131,7 @@ def plot_survey(
 
     ax.set_xlabel(r"$s$ [m]")
 
-    ax.set_ylim(-1, 1)
+    ax.set_ylim(-height_scale, height_scale)
     ax.set_yticks([])
 
     ax.set_aspect(1 / 1.618)  # golden ratio
```

Comment:
Another approach: maybe replacing this with
```suggestion
    ax.set_box_aspect(1 / 6)  # just some nice ratio on canvas
```
Could help. Originally, I mean to set the ratio of the [axes box on canvas](https://matplotlib.org/stable/api/_as_gen/matplotlib.axes.Axes.set_box_aspect.html), not the aspect based on the plotted data.

This needs matplotlib 3.3+ or 3.1+ or so (for our `requirements.txt`)

Reply:
I think this is a nice approach. I think this looks nice. Looks like matplotlib 3.3+

Reply:
Just thought about it: If going that route one could also easily make the aspect-ratio be accessible for the user and just default to 1/6
Would just need another optional argument

Reply:
Sounds great! Do you like to adjust your PR accordingly by pushing another commit to it?

>  Just thought about it: If going that route one could also easily make the aspect-ratio be accessible for the user and just default to 1/6.
> Would just need another optional argument

Yes, I like that. But what I realize usually is that there are many things one might want to change in a final plot, including labels, etc. It is hard to draw a line what to pass into plot functions that way and one gets more and more arguments in the signature.

For that reason, I usually return the matplotlib `ax` object to the user in our plot functions, so one can call any other function and manipulation on it as needed :)
```py
ax = plot_survey(...)
ax.set_box_aspect(another_value)
```

Reply:
That's actually way nicer. Will update my PR tomorrow!